docs/UserGuide.md

```diff
@@ -40,13 +40,13 @@ you into a gitGud pro in the [features](#features) section.
 For experienced users of gitGud, this guide also provides a [command summary](#command-summary) for your reference, so
```

Comment:
All of the links are broken now that the numbers have been added. Please update all of the links as well.

Reply:
Good catch. Fixing them.